doc/integrator/create_application.rst.tmpl

```diff
@@ -120,12 +120,15 @@ Simple application
 
 A simple application is an application for which no custom code is needed (Python or JavaScript).
```

Comment:
Suggestion:
add: "In contrast, a _full application_ is an application for which the all possibilities for customization are made available".